source/guides/views/custom-error-pages.md

```diff
@@ -4,20 +4,18 @@ title: Guides - Custom Error Pages
 
 # Custom Error Pages
 
-When a non successful request is returned, there are some special pages that an application presents to users.
-They have a generic graphic and some basic informations like the HTTP status code and the message.
+When an unsuccessful request is returned, there are some special pages that a Hanami application presents to users.
+These pages have a generic graphic and some basic information like the HTTP status code and the message.
 
-Hanami allows to customize them on an application basis.
-We need to create a template with the corresponding HTTP code as filename (eg. `apps/web/templates/500.html.erb`).
-
-Since now on, all the side errors will be presented using that template.
+Hanami allows us to customize them on a per-application basis.
+We just need to create a template with the corresponding HTTP code as the filename (e.g. `apps/web/templates/500.html.erb`).
+From then on, all the side errors will be presented using that template.
```

Comment:
For the record, I think “side errors” was supposed to be “server-side errors”. But you already improved that section’s wording to be more even more specific, in be0e557b8f6e4c35dc7435fb741dbf03239b5340.